contracts/src/core/EigenDACertVerifier.sol

```diff
@@ -80,45 +80,45 @@ contract EigenDABlobVerifier is IEigenDABlobVerifier {
     /**
      * @notice Verifies a blob for the base required quorums and the default security thresholds
      * @param batchHeader The batch header of the blob
-     * @param blobVerificationProof The blob verification proof for the blob
+     * @param blobInclusionInfo The inclusion proof for the blob
      * @param nonSignerStakesAndSignature The nonSignerStakesAndSignature for the blob
      */
-    function verifyBlobV2(
+    function verifyDACertV2(
         BatchHeaderV2 calldata batchHeader,
-        BlobVerificationProofV2 calldata blobVerificationProof,
+        BlobInclusionInfo calldata blobInclusionInfo,
         NonSignerStakesAndSignature calldata nonSignerStakesAndSignature
```

Comment:
Could we take this change as an opportunity to create an explicit DACert struct, both onchain and offchain, that would hold these 3 things together? Would canonicalize the name and remove any ambiguities going forward as to what a DACert is.

Reply:
Also overall feel like we should keep the `SignedBatch` semantics, even when transforming the attestation into a NonSignerStakesAndSignature (because that's just a transformation of the exact same information, meant to be consumed by the contract differently).
![image](https://github.com/user-attachments/assets/374b5eb0-368d-4220-9d6d-53ccf9e72ffd)
Perhaps we could use `SignedBatch` and `SignedBatchWithNonsigners` or something? And then a DACert would be either {SignedBatch, BlobInclusionInfo} or {SignedBatchWithNonSigners, BlobInclusionInfo} and it would be more obvious that those two things are just one eth_call away, but have the same semantic. Right now people are always confused as to how the 2 verify functions we have are related, because syntactically they look very different.